sample_rock/rockcraft.yaml

```diff
@@ -38,3 +38,8 @@ parts:
       sample-app: srv/flask/app
     prime:
       - srv/flask/app
+    permissions:
+      - owner: 2000
+        group: 2000
```

Comment:
Aren't the permissions kept from the overlay script where there's a chown ?

Reply:
The `flask-app` part is dumping files from the host filesystem into `/srv/flask/app`. Those new files are not affected by the previous `chown`

Reply:
After testing, the "owner, group" part is required.